subprojects/docs/src/main/resources/header.html

```diff
@@ -225,6 +225,7 @@ <h3 id="authoring-gradle-builds-java">Authoring JVM Builds</h3>
                     <li><a href="../userguide/java_platform_plugin.html">Java Platform Plugin</a></li>
                     <li><a href="../userguide/groovy_plugin.html">Groovy Plugin</a></li>
                     <li><a href="../userguide/scala_plugin.html">Scala Plugin</a></li>
+                    <li><a href="../userguide/toolchain_plugins.html">Toolchain Resolver Plugins</a></li>
```

Comment:
I would not put this in the `JVM Plugins` bucket. This page is not about a plugin used to build JVM projects. It is about extending Gradle to provide support for external sources for resolving Java toolchains. If we want to make that accessible from the left navigation we could consider making the existing `Toolchains for Java projects` item an expandable item with both pages: first the one about configuring toolchains, second the one about extending Gradle to provide new toolchain sources.